src/main/java/org/reactome/release/qa/common/AbstractQACheck.java

```diff
@@ -9,6 +11,7 @@
     private final static Logger logger = Logger.getLogger(AbstractQACheck.class);
 
     protected MySQLAdaptor dba;
+    protected List<String> EMPTY_SKIP_LIST = new ArrayList<>();
```

Comment:
Looks good!  Since this is meant to be used as a constant, you could have it be `final` and optionally `public`.  You could also set it to an unmodifiable list using `Collections.emptyList()`: https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyList--

Reply:
I did all except make it public -- to keep it in line with how dba is initialized

Reply:
Hmmm... using `Collections.emptyList()` might be a good idea, if you want to have a constant be an empty list.